soc/rapidsilicon/virgo/soc.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2024 Rapid Silicon.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * VIRGO SoC specific interrupt map and mask functions
+ */
+
+#include <zephyr/kernel.h>
+
+#define SOC_IRQ_MASK_OFFSET 0
+#define SOC_IRQ_MAP_OFFSET  16
+
+#define SOC_IRQ_DISABLE     0x0
+#define SOC_IRQ_ENABLE      0x1
```

Comment:
```suggestion
#define SCU_MASK_DISABLE     0x0
#define SCU_MASK_IRQ_ENABLE      0x1
```
This will tell us in the code where exactly we are enabling / disabling the IRQ